src/components/Page.jsx

```diff
@@ -1,10 +1,11 @@
-import { sbEditable } from "@storyblok/storyblok-editable";
-import Components from "./index";
+import { storyblokEditable, StoryblokComponent } from "@storyblok/react";
 
-const Page = ({ content }) => (
-	<div {...sbEditable(content)}>
-		{content?.body?.map((blok) => Components(blok))}
-	</div>
+const Page = ({ blok }) => (
+  <div {...storyblokEditable(blok)}>
+    {blok?.body?.map((nestedBlok) => (
+      <StoryblokComponent blok={nestedBlok} key={nestedBlok._uid} />
```

Comment:
I am not sure if you should not use `uuid` instead of `_uid`. Check this https://www.storyblok.com/faq/what-is-difference-between-story-id-and-uuid

@christianzoppi what do you think?

Reply:
@samuells `_uid` is the correct one in this context because nested blocks have just that property and not `uuid` which is a property of the story object

Reply:
ok 👍